config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -253,9 +253,9 @@ periodics:
             --config=ci \
             "--build=//... -//vendor/..." \
             --release=//build/release-tars \
-            --gcs=gs://kubernetes-release-dev/ci \
+            --gcs=gs://kubernetes-release-dev/ci-canary \
             --version-suffix=-bazel \
-            --publish-version=gs://kubernetes-release-dev/ci/latest-bazel.txt
+            --publish-version=gs://kubernetes-release-dev/ci/latest-bazel-canary.txt
```

Comment:
not sure what this is for now. also tagging @fejta on this one.

Reply:
`launcher.gcr.io/google/bazel` uses the latest bazel vesion, rather than a specific one.